orttraining/orttraining/python/training/ortmodule.py

```diff
@@ -213,8 +215,11 @@ def backward(ctx, *grad_outputs):
         # Get the module that flattens the output from the original module into a tuple
         self._flattened_output_module = \
             _ortmodule_output_transformation.get_flattened_output_module(self._original_module)
-        sig = signature(self._original_module.forward)
-        self._original_module_input_names = sig.parameters.keys()
+        self._original_module_parameters = signature(self._original_module.forward).parameters.values()
+        for input_parameter in self._original_module_parameters:
+            if input_parameter.kind == inspect.Parameter.VAR_KEYWORD:
+                warnings.warn("The model's forward method has **kwargs parameter which is not supported.")
```

Comment:
Shall we instead raise an exception? Since this model cannot be exported, users would not be able to use this ```ORTModule``` instance.

Reply:
I want just warning because **kwargs may be just defined in the signature but not actually used.

Reply:
I tired to add a test and even if `**kwargs` are not used, the exporter fails (I tried some possible workarounds without success).
So I'm changing this to exception, as you suggested.